src/backend/workers/go.remoteIDs/cmd/idpoller/cli_cmds/root.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * // Copyleft (ɔ) 2018 The Caliopen contributors.
+ * // Use of this source code is governed by a GNU AFFERO GENERAL PUBLIC
+ * // license (AGPL) that can be found in the LICENSE file.
+ */
+
+/*
+ * // Copyleft (ɔ) 2018 The Caliopen contributors.
+ * // Use of this source code is governed by a GNU AFFERO GENERAL PUBLIC
+ * // license (AGPL) that can be found in the LICENSE file.
+ */
+
+package cmd
+
+import (
+	. "github.com/CaliOpen/Caliopen/src/backend/workers/go.remoteIDs"
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	config     PollerConfig
+	configFile string
+	configPath string
+	verbose    bool
+	version    bool
+	RootCmd    = &cobra.Command{
+		Use:   "idpoller",
+		Short: "Remote Identities poller",
+		Long:  `idpoller is a daemon to monitor users's remote identities polls and send NATS messages accordingly`,
+		Run:   nil,
+	}
+)
+
+const __version__ = "0.1.0"
```

Comment:
Missing definition of application to build during release in devtools/package.yaml